### DISCUSSION TITLE
Get libuv static library for iOS

### DISCUSSION THREAD

**Discussion by pedro-alfonso-miris** [libuv/libuv#4394] [Q&A]
Hi,

We are trying to use libuv.a in Xcode to run an iOS destination target, but we are getting below error:

```
ld: building for 'iOS', but linking in object file (/Users/.../Documents/Workspace/.../.../Frameworks/libuv.a[2](libuv_la-fs-poll.o)) built for 'macOS'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

How can we obtain a .a library able to compile for iOS?

Thanks in advance.

**Comment by saghul** [comment]
Check the build-ios job here: https://github.com/libuv/libuv/blob/v1.x/.github/workflows/CI-unix.yml

**Comment by pedro-alfonso-miris** [comment]
Thanks a lot we will try it.

We forgot to mention, we also need to get a proper build for xrOS. Maybe trying with the same script is enough to get it?

Thoughts?...
